Web/Edubase.Services/Nomenclature/NomenclatureService.cs

```diff
@@ -43,6 +43,7 @@ public static string GetEstablishmentsPluralName(GT groupType, eTextCase textCas
                 case GT.MultiacademyTrust:
                 case GT.SingleacademyTrust:
                 case GT.SchoolSponsor:
+                case GT.SecureSingleAcademyTrust:
```

Comment:
This code references SAT - should Secure SAT be included also?

https://github.com/DFE-Digital/get-information-about-schools/blob/secure_16-19_groups/Web/Edubase.Web.UI/Areas/Groups/Views/Group/Details.cshtml#L174-L177

```csharp
                            @if (Model.GroupTypeId.OneOfThese(eLookupGroupType.MultiacademyTrust, eLookupGroupType.SingleacademyTrust, eLookupGroupType.SchoolSponsor, eLookupGroupType.Federation))
                            {
                                @OtherServicesLinks()
                            }
```

I note that there are similar references within this file also (links to external services such as financial benchmarking), but a) that file is currently diverged from the `dev` branch, and b) it unclear to me if it makes business sense for there to be e.g. a financial benchmarking lookup logic for a SAT/Secure SAT.

https://github.com/DFE-Digital/get-information-about-schools/blob/dev/Web/Edubase.Web.UI/Areas/Groups/Models/GroupDetailViewModel.cs

Reply:
I had assumed this would not be visible.  But we are having a meeting with Mark so I will check.

Reply:
It'll have to be a new story if it's needed - there're no current ACs about it, so not part of this PR is my feeling.

Reply:
I think Catherine is right on this.  Its nice to be helpful by fixing things we think should be on the story etc.  But I think we have to draw the line at things that are new.  Or the estimate we made could or should be bigger.